AmplifyPlugins/Core/AWSPluginsCoreTests/Model/GraphQLRequest/GraphQLRequestSyncCustomPrimaryKeyTests.swift

```diff
@@ -220,4 +221,65 @@ class GraphQLRequestSyncCustomPrimaryKeyTests: XCTestCase {
         XCTAssertEqual(input["orderId"] as? String, expectedInput["orderId"] as? String)
         XCTAssertEqual(input["_version"] as? Int, expectedInput["_version"] as? Int)
     }
+
+    func testDeleteMutationGraphQLRequestWithDateInPK() throws {
+        do {
+            let datetime = try Temporal.DateTime(iso8601String: "2020-01-20T08:00")
+            let customer = CustomerWithDateInPK(dob: datetime, firstName: "John", lastName: "Doe")
+            var documentBuilder = ModelBasedGraphQLDocumentBuilder(modelName: customer.modelName,
+                                                                   operationType: .mutation)
+            documentBuilder.add(decorator: DirectiveNameDecorator(type: .delete))
+            documentBuilder.add(decorator: ModelIdDecorator(model: customer))
+            documentBuilder.add(decorator: ConflictResolutionDecorator(version: 1, lastSync: nil))
+            let document = documentBuilder.build()
+            let documentStringValue = """
+                mutation DeleteCustomerWithDateInPK($input: DeleteCustomerWithDateInPKInput!) {
+                  deleteCustomerWithDateInPK(input: $input) {
+                    id
+                    createdAt
+                    dob
+                    firstName
+                    lastName
+                    updatedAt
+                    __typename
+                    _version
+                    _deleted
+                    _lastChangedAt
+                  }
+                }
```

Comment:
For conflict resolution enabled API, we care mostly about create/update/delete mutations for sync to cloud and subscriptions for initial sync process. Looks like we have the delete mutation with version tested, can we also add subscription like onCreate test as a minimum? A stretch goal would be to add test for non-conflict resolution enabled API

Reply:
+ syncQuery test as well!